frontend/src/components/Admin/SiteDeployer.tsx

```diff
@@ -30,21 +26,8 @@ const SiteDeployer: React.FC = () => {
   };
 
   useEffect(() => {
-    if (selfMember) {
-      PermissionsAPI.isAdmin()
-        .catch(() => false)
-        .then((isAdmin) => {
-          if (isAdmin || selfMember.role === 'lead') {
-            loadPullRequests();
-          } else {
-            Emitters.generalError.emit({
-              headerMsg: 'Access Denied',
-              contentMsg: `Insufficient permissions.`
-            });
-          }
-        });
-    }
-  }, [selfMember]);
```

Comment:
We guard at the stage of navigation cards, so non-admin won't even enter the page.

Reply:
Are nav cards that lead to admin pages just not shown to non-admin users now?

Reply:
Yep. See changes in frontend/src/pages/admin/index.tsx